src/operation.js

```diff
@@ -10,6 +10,30 @@ import buildResponse from './response';
 import Validator from './validation';
 
 
+function getRetries(request) {
+    if (
+        includes(
+            [
+                'post',
+                'patch',
+                'put',
+                'delete',
+            ],
+            lowerCase(request.method),
+        )
+    ) {
+        // Mutations will be retried on explicit initiation,
```

Comment:
Can we somehow "force" them to retry?
If not here - let's add a test to show what's the right way to retry a mutation operation.

Reply:
I'm not wholly sure what you mean. The "force" will be the client themselves making the decision to retry, given that it isn't "safe" by default. In the same vein, I added a test saying that mutations won't be retried by the library, but a client always free to re-do it themselves.